### DISCUSSION TITLE
Can't log in to Crashplan since upgrading to Container Manager on Synology

### DISCUSSION THREAD

**Discussion by NJSailRacer** [jlesage/docker-crashplan-pro#447] [Q&A]
Has anyone managed to get Crashplan running under the new Synology container manager?  It worked great under Docker, but now I can't log in to the Crashplan app either via browser or VNC.  Firewall is off. Log is attached.
[Crashplan-pro1.csv](https://github.com/jlesage/docker-crashplan-pro/files/13527041/Crashplan-pro1.csv)

**Comment by jlesage** [comment]
Did you verify in container settings that port `5800` is correctly mapped ?

**Reply by NJSailRacer** [reply]
I think so.  Does this look correct?
![image](https://github.com/jlesage/docker-crashplan-pro/assets/66130525/da803372-eab0-4918-93e0-8849ec0f89aa)

**Reply by NJSailRacer** [reply]
Is there any other information that I can provide to help in troubleshooting?

**Comment by jlesage** [comment]
So the URL yo use to connect is something like `http://<nas ip>:49165` ?

**Reply by NJSailRacer** [reply]
Yes, my internal IP is http://192.168.1.224:49165

This worked fine using Docker under Synology v6.  Now I get Connection Refused using Containers under Synology v7

**Comment by jlesage** [comment]
To make sure the container is really configured correctly, could you provide the output of the command `docker inspect <container name>` ?

**Reply by NJSailRacer** [reply]
![image](https://github.com/jlesage/docker-crashplan-pro/assets/66130525/bc3b6fbf-175b-4d8d-994c-959390c20d66)

**Reply by NJSailRacer** [reply]
![image](https://github.com/jlesage/docker-crashplan-pro/assets/66130525/0c8b1a28-d629-4c42-a63c-fa90262385d4)
Does this look correct?

**Comment by jlesage** [comment]
It seems that you need to run the command with `sudo`: `sudo docker inspect <container name>`.  Please copy-paste the result.

**Reply by NJSailRacer** [reply]
[Crashplan-pro1.txt](https://github.com/jlesage/docker-crashplan-pro/files/13625427/Crashplan-pro1.txt)
Thanks for the tip. That worked.  Please find attached the inspect output.

**Reply by jlesage** [reply]
Everything looks good.  I'm not sure why you can't access the page.  Since the problem started after Synology upgrade, you may want to seek help from them as well ?

**Reply by NJSailRacer** [reply]
Thanks.  Is there anyone else running this container under Synology 7?
